chaincfg/params.go

```diff
@@ -262,7 +263,7 @@ var MainNetParams = Params{
 	Name:        "mainnet",
 	Net:         wire.MainNet,
 	DefaultPort: "8333",
-	DNSSeeds: []DNSSeed{
+	DNSSeeds: []DNSSeed{ // TODO change this to proper dns seed
```

Comment:
Just a note, I run seed.bchd.cash, and as we finish changes will update that codebase.